tests/integration/backend_test.py

```diff
@@ -834,9 +834,18 @@ def test_wasm(
 @pytest.mark.parametrize(
     "authenticated_quum_backend_qa", [{"device_name": "H1-1E"}], indirect=True
 )
+@pytest.mark.parametrize(
+    "language",
+    [
+        Language.QASM,
+        Language.QIR,
+        Language.PQIR,
+    ],
+)
 @pytest.mark.timeout(120)
 def test_wasm_costs(
     authenticated_quum_backend_qa: QuantinuumBackend,
+    language: Language,
```

Comment:
Should use this in the call to `cost()`?

Reply:
Yes, done